### DISCUSSION TITLE
SSL_ERROR_INTERNAL_ERROR_ALERT

### DISCUSSION THREAD

**Discussion by ANGdesarrollo** [vercel/community#4530] [Help] [answered]
### Summary

Hello,

I'm experiencing an issue with SSL_ERROR_INTERNAL_ERROR_ALERT when attempting to access https://www.carnivorasargentinas.com/. I configured the domain on Vercel approximately 2 hours ago. The domain was originally registered with Hostinger. I can successfully access the site using my mobile connection with 4G, but I encounter the SSL_ERROR_INTERNAL_ERROR_ALERT when attempting to access it via any Wi-Fi network.

Is there anyone who can assist me in resolving this issue?

Thank you very much.

### Steps to Reproduce

Enter to https://www.carnivorasargentinas.com/ via WIFI.

Enter to https://www.carnivorasargentinas.com/ via 4g mobile.

**Comment by amyegan** [comment, marked as answer]
Hi @ANGdesarrollo. Your domain is working correctly on Wi-Fi for me. Are you still having trouble with it?

Remember that [DNS propagation](https://vercel.com/docs/projects/domains/working-with-dns#dns-propagation) is not instant, and it can take some time for records to update throughout the global network of servers. Vercel automatically tries to [generate an SSL certificate](https://vercel.com/docs/projects/domains/working-with-ssl#), but the certificate validation request will only succeed after DNS records are added and propagated.

**Comment by ANGdesarrollo** [comment]
Hi, it is still not working on my wifi company, but a lot of people can see
it, its a problem of my wifi company that is not getting the updated DNS.

I will wait 2 more days to see what happens. Thank you very much.
